server/eventlog_test.go

```diff
@@ -690,6 +690,38 @@ func TestParseSecureBootState(t *testing.T) {
 		if !contains3PUEFI || !containsWinProdPCA {
 			t.Error("expected to see both WinProdPCA and ThirdPartyUEFI certs")
 		}
+
+		if len(msState.GetSecureBoot().GetPk().GetHashes()) != 0 {
+			t.Error("found hashes in pk")
+		}
+		for _, cert := range msState.GetSecureBoot().GetPk().GetCerts() {
+			switch c := cert.GetRepresentation().(type) {
+			case *attestpb.Certificate_Der:
+				if !bytes.Equal(c.Der, GceDefaultPKCert) {
+					t.Errorf("found pk cert der: %+v expecing: %+v", c.Der, GceDefaultPKCert)
+				}
+			case *attestpb.Certificate_WellKnown:
+				t.Errorf("Unexpected well-known cert in pk")
+			default:
+				t.Errorf("Unexpected cert representation type in pk")
+			}
+		}
+
+		if len(msState.GetSecureBoot().GetKek().GetHashes()) != 0 {
+			t.Error("found hashes in kek")
+		}
+		for _, cert := range msState.GetSecureBoot().GetKek().GetCerts() {
+			switch c := cert.GetRepresentation().(type) {
+			case *attestpb.Certificate_Der:
+				if !bytes.Equal(c.Der, MicrosoftKEKCA2011Cert) {
```

Comment:
nit: it would be nice to update https://github.com/google/go-tpm-tools/blob/7fe225f6e5a24688cd95f8866fd855a3a10e0ef3/server/eventlog.go#L418 with the MS KEK

Reply:
Done.
And as discussed offline, I also added GceDefaultPKCert to this list of well known certificates.